src/index.html

```diff
@@ -5,8 +5,363 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com" crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#Home">
+        <img
+          src="/images/logo.png"
+          alt="MOYO logo"
+          class="header__logo"
+        >
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a class="header__link is-active" href="#apple">Apple</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#samsung">Samsung</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#smartphones">Smartphones</a>
+          </li>
+          <li class="header__item">
+            <a
+              class="header__link"
+              href="#computers"
+              data-qa="nav-hover"
+            >
+              laptops & Computers
+            </a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#gadgets">Gadgets</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#tablets">Tablets</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#photo">Photo</a>
+          </li>
+          <li class="header__item">
+            <a class="header__link" href="#video">Video</a>
+          </li>
+        </ul>
```

Comment:
Between sibling elements it would be better to leave one empty line for clearer code readability.

Reply:
@amashutikov so?